go-controller/pkg/ovn/multipolicy_test.go

```diff
@@ -337,21 +363,34 @@ var _ = ginkgo.Describe("OVN MultiNetworkPolicy Operations", func() {
 				Items: nads,
 			},
 		)
-		ocInfo, ok := fakeOvn.secondaryControllers[secondaryNetworkName]
-		gomega.Expect(ok).To(gomega.Equal(true))
 
-		for _, testPod := range pods {
-			testPod.populateLogicalSwitchCache(fakeOvn)
-		}
 		var err error
+		if config.OVNKubernetesFeature.EnableInterconnect {
+			for _, node := range nodes {
+				transistSwitchPortName := ovntypes.TransitSwitchToRouterPrefix + node.Name
+				err := libovsdb.CreateTransitSwitchPortBindings(fakeOvn.sbClient, ovntypes.TransitSwitch, transistSwitchPortName)
+				gomega.Expect(err).NotTo(gomega.HaveOccurred())
+			}
+		}
+		if nodes != nil {
+			err = fakeOvn.controller.WatchNodes()
+			gomega.Expect(err).NotTo(gomega.HaveOccurred())
```

Comment:
nit: same here. I'd inline these.

Reply:
Doesn't really make it more readable for me, will leave as is.